industrial_ci/mockups/testpkg_broken_install/script/sample_talker.py

```diff
@@ -0,0 +1,10 @@
+# Taken from http://wiki.ros.org/rospy/Overview/Publishers%20and%20Subscribers
```

Comment:
I see, but we need something for [the `install` rule](https://github.com/ros-industrial/industrial_ci/pull/177/files/c1b8bbfddd5f307b207c4bdc7a02c09d76bf5449#diff-2b29029d69488436374c1e649c2236b7R9) (although a directory without any content should work), so let us just leave it for now.